extensions/geode-modules/src/main/java/org/apache/geode/modules/session/bootstrap/AbstractCache.java

```diff
@@ -46,26 +46,31 @@
 
   private static final DateFormat FORMAT = new SimpleDateFormat("yyyy-MM-dd");
 
-  protected static final String DEFAULT_LOG_FILE_NAME =
+  private static final String DEFAULT_LOG_FILE_NAME =
       RegionHelper.NAME + "." + FORMAT.format(new Date()) + ".log";
 
-  protected static final String DEFAULT_STATISTIC_ARCHIVE_FILE_NAME = RegionHelper.NAME + ".gfs";
+  private static final String DEFAULT_STATISTIC_ARCHIVE_FILE_NAME = RegionHelper.NAME + ".gfs";
 
-  protected static final float DEFAULT_EVICTION_HEAP_PERCENTAGE =
+  private static final float DEFAULT_EVICTION_HEAP_PERCENTAGE =
       LocalRegion.DEFAULT_HEAPLRU_EVICTION_HEAP_PERCENTAGE;
 
-  protected static final float DEFAULT_CRITICAL_HEAP_PERCENTAGE =
+  private static final float DEFAULT_CRITICAL_HEAP_PERCENTAGE =
       ResourceManager.DEFAULT_CRITICAL_PERCENTAGE;
 
-  protected static final Logger LOGGER = LoggerFactory.getLogger(AbstractCache.class);
+  // TODO: DistributionConfig is internal, change these fields when the attributes are public.
```

Comment:
Not sure if this TODO was for you or the next person

Reply:
Forgot to remove the comment, thanks.